src/components/Equipment.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import ItemList from './ItemList';
+import torches from './../assets/image/ekwipunek/torches.png';
+
+const Equipment = () => {
+  const [items, setItems] = useState([]);
```

Comment:
Tutaj typy normalnie działają? Jak iterujesz po itemsach to podpowiada Ci VSC property?